src/client/components/forms/userCollection.js

```diff
@@ -0,0 +1,306 @@
+			this.handleAddCollaborator();
+		}
+
+		const privacyOptions = ['Private', 'Public'].map((option) => ({
+			name: option
+		}));
+		const entityTypeOptions = ['Author', 'Work', 'Edition', 'Edition-Group', 'Publisher'].map((entity) => ({
+			name: entity
+		}));
+		const initialName = this.state.collection.name;
+		const initialDescription = this.state.collection.description;
+		const initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
+		const initialType = this.state.collection.entityType;
+		const {errorText} = this.state;
+		const errorAlertClass =
+			classNames('text-center', 'margin-top-1', {hidden: !errorText});
+		const submitLabel = this.props.collection.name ? 'UPDATE' : 'CREATE';
+
+		/* eslint-disable react/jsx-no-bind */
+		return (
+			<Grid>
+				<h1>Create a collection</h1>
+				<Row>
+					<Col md={12}>
+						<p className="lead">Create Your Collection</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col
+						id="collectionForm"
+						md={8}
+						mdOffset={2}
+					>
+						<form
+							className="form-horizontal"
+							onSubmit={this.handleSubmit}
+						>
+							<CustomInput
+								defaultValue={initialName}
+								label="Name"
+								ref={(ref) => this.name = ref}
+								type="text"
+							/>
+							<CustomInput
+								defaultValue={initialDescription}
+								label="Description"
+								ref={(ref) => this.description = ref}
+								type="textarea"
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialType}
+								disabled={!this.props.canEditType}
+								idAttribute="name"
+								label="Entity Type"
+								labelAttribute="name"
+								options={entityTypeOptions}
+								placeholder="Select Entity Type"
+								ref={(ref) => this.entityType = ref}
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialPrivacy}
+								idAttribute="name"
+								label="Privacy"
+								labelAttribute="name"
+								options={privacyOptions}
+								placeholder="Select Privacy"
+								ref={(ref) => this.privacy = ref}
+							/>
+							{
+								this.state.collection.collaborators.map((collaborator, index) => (
+									<div key={collaborator.id}>
+										<EntitySearchFieldOption
+											instanceId="collaboratorSearchField"
+											label="Select Collaborator"
+											name="editor"
+											type="editor"
+											value={collaborator}
+											onChange={(newCollaborator) => this.handleChangeCollaborator(newCollaborator, index)}
+										/>
+										<Button
+											bsSize="small"
+											bsStyle="primary"
+											type="button"
+											onClick={() => this.handleRemoveCollaborator(index)}
+										>
+											Remove
+										</Button>
+									</div>
+								))
+							}
+							<div className="text-center">
+								<Button
+									type="button"
+									onClick={this.handleAddCollaborator}
+								>
+									Add Collaborator
```

Comment:
Currently, selecting a user from the dropdown already selects them without having to press this button correct?
Meaning this button is to add an extra empty collaborator?

If so, then the button should read "Add another collaborator", maybe with a little "plus" sign icon